.circleci/config.yml

```diff
@@ -229,6 +229,25 @@ jobs:
       - store_artifacts:
           path: /home/circleci/project/target/inspection-results
 
+  no-merge-commits:
+    docker:
+      - image: cimg/base:2022.10
+    steps:
+      - checkout
+      - run:
+          name: Check for merge commits
+          command: |
+            MERGE_COMMITS=$(git rev-list --merges master.."$CIRCLE_SHA1")
```

Comment:
I think @romani might ask this to also be in a validation or similar file.

Reply:
Done